### ISSUE TITLE
Brakeman crashes while printing warning for Ruby EOL 2.6

### ISSUE THREAD

**Issue by klazuka** [presidentbeef/brakeman#1671]
Earlier this morning we started seeing Brakeman crash with error:
```
gems/brakeman-5.2.0/lib/brakeman/warning.rb:251:in `sprintf': can't convert nil into Integer (TypeError)
```

I traced the error and found that it crashed while reporting a warning for `CheckEOLRuby`. The root cause appears to be that the warning symbol `:pending_eol_ruby` was introduced recently [here](https://github.com/presidentbeef/brakeman/blob/8ccf86b2fddcb363481618561aa1afc808b9a259/lib/brakeman/checks/eol_check.rb#L34) in #1660, but never had a numeric code registered.

I was able to fix the problem by editing https://github.com/presidentbeef/brakeman/blob/main/lib/brakeman/warning_codes.rb to assign `:pending_eol_ruby` a numeric code.

**Comment by presidentbeef** [comment]
Ah, embarrassing. I'll get a fix out ASAP.

**Comment by presidentbeef** [comment]
Fixed with 5.2.1. Thanks and apologies!

**Comment by klazuka** [comment]
@presidentbeef no problem. Thanks for the quick fix!
